src/drive/web/modules/drive/Toolbar/components/StartScanner.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import { ActionMenuItem } from 'cozy-ui/transpiled/react/ActionMenu'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import { getTracker } from 'cozy-ui/transpiled/react/helpers/tracker'
+import { SCANNER_UPLOADING } from 'cozy-scanner'
+import { ScannerContext } from './ScanWrapper'
+
+const StartScanner = ({ t }) => {
+  const { status, online, startScanner } = useContext(ScannerContext)
+
+  const offlineMessage = () => {
+    return alert(t('Scan.error.offlinee'))
```

Comment:
Is it expected to use window.alert instead of the Alerter ?

Reply:
Apparently yes! This code only runs on mobile and the native alerts look good and give the app a more native feel.